Tools/autotest/common.py

```diff
@@ -445,6 +445,39 @@ def set_throttle_zero(self):
         else:
             self.set_rc(3, 1000)
 
+    def set_output_to_max(self, chan):
+        """Set output to max with RC Radio taking into account REVERSED parameter"""
+        is_reversed = self.get_parameter("RC%u_REVERSED" % chan)
+        out_max = self.get_parameter("RC%u_MAX" % chan)
+        out_min = self.get_parameter("RC%u_MIN" % chan)
+        if is_reversed == 0:
+            self.set_rc(chan, out_max)
+        else:
+            self.set_rc(chan, out_min)
+
+    def set_output_to_min(self, chan):
+        """Set output to min with RC Radio taking into account REVERSED parameter"""
+        is_reversed = self.get_parameter("RC%u_REVERSED" % chan)
+        out_max = self.get_parameter("RC%u_MAX" % chan)
+        out_min = self.get_parameter("RC%u_MIN" % chan)
+        if is_reversed == 0:
+            self.set_rc(chan, out_min)
+        else:
+            self.set_rc(chan, out_max)
+
+    def get_arming_rudder(self):
```

Comment:
This should simply return the value of `RCMAP_RUDDER`, shouldn't it?

Reply:
The issues is that RCMAP_RUDDER doesn't exist ... the channel is different for rover and plane/copter. Anyway I change to use the RCMAP !